packages/diagrams/frontend/sirius-components-diagrams/src/renderer/palette/tool-list/usePaletteEntryTooltip.ts

```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { UsePaletteEntryTooltipValue } from './usePaletteEntryTooltip.types';
+
+export const usePaletteEntryTooltip = (): UsePaletteEntryTooltipValue => {
```

Comment:
I think this should be removed to use default value and then we could override the theme to set these. 
What do you think ? @sbegaudeau